packages/elements/src/datetime-field/index.ts

```diff
@@ -537,7 +537,7 @@ export class DatetimeField extends TextField {
    * @param parts The list of parts
    * @returns {void}
    */
-  protected selectPart (index = 0, parts: DateTimeFormatPart[]): void {
+  protected selectPart (index: number, parts: DateTimeFormatPart[]): void {
```

Comment:
The `index` should be an optional parameter and move to the last to keep the parameters consistent with the method name.